File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String rs = my_string.replaceAll("[^0-9]", "");
        
        for(int i=0; i<rs.length(); i++){
          answer += Integer.parseInt(String.valueOf(rs.charAt(i)));
            
        }
        return answer;
    }
}

File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.py
def solution(my_string):
    answer = 0
    #방법1 : for문와 if문
    # for i in my_string:
    #     if i.isdigit():
    #         answer += int(i)
    
    #방법2 : sum함수를 활용, 한줄 작성
    answer = sum(int(i) for i in my_string if i.isdigit())
    return answer